### PR TITLE
Update actions

### LINKED ISSUE #1009: Setup github actions for react library releases

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1009

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -31,7 +25,7 @@ module.exports = {
   },
   settings: {
     react: {
-      version: reactVersion,
+      version: '^17.0.1',
```
Comment (wenche) [inline .eslintrc.js:28]: Wish the original approach worked 🙈

**Review by wenche — CHANGES_REQUESTED** [review]
As mentioned in the standup today, I suggest we update `RELEASE.md` file according to this new regime as well 👍

File: .eslintrc.js (hunk 1 above)
Comment (mimarz) [reply .eslintrc.js:28]: sparse-checkout... 🙈

File: RELEASE.md [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+
+### Prepare
+
+This can be done from develop or seperate release branches (before merging to develop)
+
+1. Change the version-number in accordance with the magnitude of change.
+   1. If its not a `@latest` release, the type should be "dashed", for example; `0.0.0-beta` or `0.0.0-dev.20210101`
+   2. We try to follow [semver](https://semver.org/) but are holding back major v1 until we feel its ready.
+2. Update the changelog
+3. Add a release in github
```
Comment (wenche) [inline RELEASE.md:13]: Elaborate a bit here?

**Review by wenche — CHANGES_REQUESTED** [review]
Looking good as far as I can see, but I guess we need to try it for ourselves to see how it works. 

Good job on automation 👍

File: RELEASE.md (hunk 2 above)
Comment (wenche) [reply RELEASE.md:13]: On the `3. Add a release in github`

Comment (mimarz) [reply RELEASE.md]: Done!

**Review by wenche — APPROVED** [review]
🥳

**Pull request merged**